docs/src/views/ColorPalette.tsx

```diff
@@ -0,0 +1,92 @@
+import { Divider, Grid, Header, Provider, Segment, Text } from '@stardust-ui/react'
+import * as faker from 'faker'
+import * as _ from 'lodash'
+import * as React from 'react'
+
+import ColorBox, { colorBoxStyles, colorBoxVariables } from 'docs/src/components/ColorBox'
+import ColorVariants, { colorVariantsStyles } from 'docs/src/components/ColorVariants'
+import DocPage from 'docs/src/components/DocPage/DocPage'
+import ProviderConsumer from 'src/components/Provider/ProviderConsumer'
+import { unsupportedColorVariants } from '../../../src/lib'
+
+const naturalColors = ['blue', 'green', 'grey', 'orange', 'pink', 'purple', 'teal', 'red', 'yellow']
+
+const red = unsupportedColorVariants('red', {
+  500: 'yellow',
+})
+
+const ColorPalettePage = () => (
+  <Provider
+    theme={{
+      componentStyles: {
+        ColorBox: colorBoxStyles,
+        ColorVariants: colorVariantsStyles,
+        Grid: {
+          root: {
+            gridColumnGap: '.3rem',
+            gridRowGap: '.3rem',
+          },
+        },
+      },
+      componentVariables: {
+        ColorBox: colorBoxVariables,
+      },
+    }}
+  >
+    <ProviderConsumer
+      render={({ siteVariables: { colors } }) => (
+        <DocPage description="An RFC for color palette" title="Color Palette">
+          <Header as="h2">Natural colors</Header>
+          <p>TODO: Describe</p>
+
+          <Grid columns={2}>
+            {_.map(['black', 'white'], color => (
+              <ColorBox key={color} name={color} rounded size="big" value={colors[color]} />
+            ))}
+          </Grid>
+          <p>COLOR: {red[400]}</p>
+          <p>COLOR: {red[500]}</p>
+          <p>COLOR: {red[600]}</p>
+          <p>COLOR: {red[400]}</p>
+
+          <Divider />
+
+          <Grid columns={3}>
+            {_.map(naturalColors, color => (
+              <ColorBox key={color} name={color} rounded value={colors[color][500]} />
+            ))}
+          </Grid>
+
+          <Header as="h2">Contextual colors</Header>
+          <p>TODO: Describe</p>
+
+          <Grid columns={2}>
+            {_.map(['primary', 'secondary'], color => (
+              <ColorBox key={color} name={color} rounded size="big" value={colors[color][500]} />
+            ))}
+          </Grid>
+
+          <Header as="h2">Text colors</Header>
+          <p>TODO: Describe</p>
+
+          <Segment>
+            {_.map(colors.text, color => (
+              <Text as="p" key={color} style={{ color }}>
```

Comment:
`Text` has `display: inline-block` - that results in multiple colors displayed on a single line which can be difficult to understand.
![image](https://user-images.githubusercontent.com/9615899/48997891-c5eba580-f151-11e8-8417-0ad721cc5996.png)

Reply:
I changed `Text` to a simple `p` 👍